src/pages/dataSetNotificationTemplates/form/getInitialValuesFromTemplate.ts

```diff
@@ -0,0 +1,100 @@
+export interface DataSet {
+    id: string
+    name: string
+    displayName: string
+}
+
+export interface DataSetNotificationTemplate {
+    id: string
+    code: string
+    name: string
+    displayName: string
+    subjectTemplate: string
+    messageTemplate: string
+    notificationRecipient: string
+    dataSetNotificationTrigger: string
+    relativeScheduledDays: number
+    sendStrategy: string
+    deliveryChannels: string[]
+    recipientUserGroup?: { id: string }
+    dataSets: Array<{ id: string }>
+}
+
+export type DataSetNotificationFormValues = Omit<
+    DataSetNotificationTemplate,
+    'relativeScheduledDays' | 'recipientUserGroup'
+> & {
+    relativeScheduledDays: string
```

Comment:
I don't think we need to handle this as a string? We could handle the "string" output of the field, and parse it to an int on the form-level through final-forms `parse` property to `FieldRFF`.